tools/osx-codesign.sh

```diff
@@ -1,17 +1,18 @@
-#!/bin/bash
+#!/bin/sh
 
 set -x
 set -e
 
-if [ "X$SIGN" == "X" ]; then
-  echo "No SIGN environment var.  Skipping codesign." >&2
+# shellcheck disable=SC2154
```

Comment:
I don't think this disable directive is necessary?

Reply:
I'm getting this warning when I remove this line:

```console
$ shellcheck --shell=sh --severity=info --enable=all tools/osx-codesign.sh

In tools/osx-codesign.sh line 6:
[ -z "$SIGN" ] && \
      ^---^ SC2154: SIGN is referenced but not assigned.
```

Reply:
I'm using ShellCheck 0.7.1 btw, that may be a version-specific behavior.